### ISSUE TITLE
ERROR: LibrarySync thread crashed. Error message: 'module' object has no attribute 'ParseError'

### ISSUE THREAD

**Issue by digger18** [croneter/PlexKodiConnect#320]
Running SPMC 16.6 on Nvidia Shield TV.
All worked perfect till this morning.

This is the traceback:

Traceback (most recent call last):
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/librarysync.py", line 1399, in run
                                                self.run_internal()
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/librarysync.py", line 1512, in run_internal
                                                librarySync = fullSync()
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/utils.py", line 969, in wrapper
                                                result = func(*args, **kwargs)
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/librarysync.py", line 266, in fullSync
                                                if self._fullSync() is False:
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/librarysync.py", line 284, in _fullSync
                                                sourcesXML()
                                              File "/storage/emulated/0/Android/data/com.semperpax.spmc16/files/.spmc/addons/plugin.video.plexkodiconnect/resources/lib/utils.py", line 711, in sourcesXML
                                                except etree.ParseError:
                                            AttributeError: 'module' object has no attribute 'ParseError'

I have reset PKC and Artwork data and even updated PMS to the latest version, but nothing helps.

My spmc.log is here:
https://pastebin.com/zfNGuAmb

Thank you for this lovely project!

**Comment by croneter** [comment]
PKC cannot parse a Plex answer. Try rescanning all your Plex libraries using Plex web to let Plex rebuild its database entries

**Comment by digger18** [comment]
I have done that but it did not help unfortunately.
Also tried to re-install PKC but that did not help also.

I was so frustrated so I decided to remove SPMC entirely and switch to a fresh Kodi v17.3 from the Play Store.
Installed PKC on it and now it's all working again.

This morning my SPMC settings got reset after I had to force shutdown my Shield a couple of times, so I guess it has do to with it.

Thanks for your addon and support!
